nbclean/clean.py

```diff
@@ -55,7 +58,7 @@ def clear(self, kind, tag=None):
         kwargs = {key: key in kind for key in ALLOWED_KINDS}
 
         # See if the cell matches the string
-        pre = ClearCells(tag=str(tag), **kwargs)
+        pre = ClearCells(tag=str(tag), search_text=str(search_text), **kwargs)
```

Comment:
why the `str`?

Reply:
Reading the below traitlets code I suspect it is to make sure the types match or something?

Reply:
yep mostly it's just to ensure the correct type...if you can think a more elegant way to do it, lemme know :-)

Reply:
You mean outside of not using traitlets :p ?